src/process_hru_shapes.R

```diff
@@ -4,7 +4,7 @@ library(lwgeom)
 proj_string <- 4326
 
 gfdb <- "cache/GF_nat_reg.gdb"
-hru_reduced <- read_sf(gfdb, "nhru")  %>% 
```

Comment:
Just curious, is this an equiavlet field?

Reply:
oh woops, for some reason the gdb i have has a different name, but no, the one on the server is nhru, will change back!

Reply:
Maybe that is Roland's newer fabric or something that you have locally?

Reply:
ah, that could very well be actually! makes sense based on the date I downloaded it